superchain/configs/sepolia/metal.yaml

```diff
@@ -17,4 +18,5 @@ genesis:
     number: 0
   l2_time: 1708129620
 
-delta_time: 1708385400   # Mon Feb 19 2024 23:30:00
\ No newline at end of file
+delta_time: 1708385400 # Tue 16 Jan 2024 15:30:00 UTC
+ecotone_time: 1708534800 # Thu 18 Jan 2024 03:00:00 UTC
```

Comment:
Something is wrong here. I get
```
date -u -r 1708385400
Mon Feb 19 23:30:00 UTC 2024

date -u -r 1708534800
Wed Feb 21 17:00:00 UTC 2024
```

Reply:
fixed here [985db7b](https://github.com/ethereum-optimism/superchain-registry/pull/233/commits/985db7bec8e2f944659500928e1c7f34b148229e)